Formula/gcc.rb

```diff
@@ -74,33 +52,33 @@ def version_suffix
     sha256 "863957f90a934ee8f89707980473769cff47ca0663c3906992da6afb242fb220"
   end
 
+    # Use -headerpad_max_install_names in the build,
+    # otherwise lto1 load commands cannot be edited on El Capitan
+    if MacOS.version == :el_capitan
+      patch do
+        url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/gcc/7.1.0-headerpad.patch"
```

Comment:
Should be `https://raw.githubusercontent.com/Homebrew/formula-patches/32cf103/gcc/7.1.0-headerpad.patch`